iocore/net/SSLUtils.cc

```diff
@@ -544,7 +547,64 @@ ssl_context_enable_ecdh(SSL_CTX *ctx)
 
   return ctx;
 }
+static ssl_ticket_key_block *
+ssl_create_ticket_keyblock(const char *ticket_key_path)
```

Comment:
Yes most of the code are duplicated from the ssl_context_enable_tickets, but in this function I am just creating the key block  not enabling session ticket for any particular context, So I decided to make a separate function. What we can do is have ssl_context_enable_ticket call ssl_create_ticket_keyblock. That way the code will look cleaner